get_started_with_membrane/06_flow_control.md

```diff
@@ -0,0 +1,41 @@
+# Flow Control
+
+Once we have linked the elements together, there comes a moment when they can start sending buffers through the pads. However, how to control the flow of buffers between elements?
+
+Membrane solves this problem using its own backpressure mechanism and demands.
+
+## Types of `flow_control`
+
+When defining a pad od an element, you can pass one of 3 values in the `flow_control` field: `auto`, `push`, or `manual`. They determine how the backpressure mechanism works on the pad they relate to.
+
+- input pad with `flow_control: :push` - an element with such a pad must always be ready to receive a new buffer on that pad. The output pad linked with such a input pad must also have `flow control` set to `:push`.
```

Comment:
I'd mention that it's usually not advised to use push input, as it's prone to overflow